### PR TITLE
fix: Minor edits to test generating a preview from a forked repo

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

The main point of this PR is to test running workflows from forks.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-1018/documentation/architecture_and_concepts/Smart_Contracts/cairo-and-sierra/

### Check List

- [x] Changes have been done against master branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1018)

### CODE REVIEW

**Review by xiaolou86 — APPROVED** [review]

**Comment by xiaolou86** [comment]
Awesome! bro!

**Comment by xiaolou86** [comment]
haha, it's my first review ^_^

**Comment by xiaolou86** [comment]
But I don't know why the deploy-review job failed. It looks like something about the gh-pages branch.

**Comment by stoobie** [comment]
Thanks, @xiaolou86 :)

It failed because I submitted it from a fork. That's what I was testing. Even though it was me, the fact that it was from a fork made it fail.

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/cairo-and-sierra.adoc [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [id="sierra"]
 = Cairo and Sierra
 
-Before xref:starknet_versions:upcoming_versions.adoc[Starknet Alpha v0.11.0] a developer would write contracts in Cairo 0 and compile them locally to Cairo assembly (or Casm for short).
-Next, the developer would submit the compilation output, the xref:Smart_Contracts/contract-classes.adoc[contract class], to the Starknet sequencer via a `DECLARE` transaction.
+Before xref:starknet_versions:upcoming_versions.adoc[Starknet Alpha v0.11.0] you would write contracts in Cairo 0 and compile them locally to Cairo assembly (Casm).
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/cairo-and-sierra.adoc:4]:
```suggestion
Before xref:starknet_versions:upcoming_versions.adoc[Starknet Alpha v0.11.0] you would write contracts in Cairo 0 and compile them locally to Cairo assembly (Casm).

Snoopy
```

**Comment by stoobie** [comment]
@xiaolou86 Looks like your fixes work!

**Pull request closed without merging**